config/config-package.py

```diff
@@ -177,78 +166,68 @@ def cfg_option(self, section, name, default=DEFAULT):
     def linting_yml(self):
         workflows = self.path / '.github' / 'workflows'
         workflows.mkdir(parents=True, exist_ok=True)
-
-        self.copy_with_meta(
-            'linting.yml.j2',
-            workflows / 'linting.yml',
-            self.config_type,
-        )
+        self.copy_with_meta('linting.yml.j2', workflows / 'linting.yml')
 
     def editorconfig(self):
-        self.copy_with_meta(
-            'editorconfig',
-            self.path / '.editorconfig',
-            self.config_type
-        )
+        self.copy_with_meta('editorconfig', self.path / '.editorconfig')
 
     def lint_requirements(self):
-        self.copy_with_meta(
-            'lint-requirements.txt.j2',
-            self.path / 'lint-requirements.txt',
-            self.config_type
-        )
+        self.copy_with_meta('lint-requirements.txt.j2')
 
     def pyproject_toml(self):
-        self.copy_with_meta(
-            'pyproject.toml.j2',
-            self.path / 'pyproject.toml',
-            self.config_type
-        )
+        self.copy_with_meta('pyproject.toml.j2')
 
     def tox(self):
-        self.copy_with_meta(
-            'tox.ini.j2',
-            self.path / 'tox.ini',
-            self.config_type,
-            dist_path=self.distribution_path
-        )
+        self.copy_with_meta('tox.ini.j2')
+
+    def news_entry(self):
+        news = self.path / 'news'
+        news.mkdir(parents=True, exist_ok=True)
+
+        destination = self.path / f'1.internal.{get_commit_id}'
```

Comment:
I did an initial fix for the new snippet, so you get something like `news/1.internal.e10f3dfc` and the relative path gets returned. But I will leave the rest for you for now.

Reply:
Thanks, I've been a bit lazy/doing it while doing other stuff, so sorry for not checking my code properly before pushing it 😅 

With the changes you did, I don't see that there is more to be done, or I'm missing something? (I have a bit of a cold 🤧 so I might not be able to focus much 😅 )

Reply:
The problem still is that `towncrier build` will fail with a `ValueError` when there are two such snippets, and I think this can happen.

With towncrier 22.12 you can use `+commit-id.internal` to have a snippet that will not be linked to an issue, so in our case the `(#1)` will not be added. See https://github.com/twisted/towncrier/pull/428. You can call `towncrier create +.internal` to create a random id, but we can use the commit id like you do.
Coredev 6 uses use this towncrier version, coredev 5.2 is stuck with an older version.

Reply:
I have merged master in here, and have updated the code to write `news/commit-id.internal`. Do you agree with this?

Sample:

```
$ ../../bin/towncrier build --draft --version=x
Loading template...
Finding news fragments...
Rendering news fragments...
Draft only -- nothing has been written.
What is seen below is what would be written.

x (2023-02-09)
--------------

Internal:


- Update configuration files
  [plone devs] (7067c6e4)
```